runtime/commonMain/src/kotlinx/serialization/internal/PluginHelperInterfaces.kt

```diff
@@ -9,6 +9,7 @@ import kotlinx.serialization.*
 @InternalSerializationApi
 public interface GeneratedSerializer<T> : KSerializer<T> {
     fun childSerializers(): Array<KSerializer<*>>
+    fun typeParametersSerializers(): Array<KSerializer<*>> = arrayOf()
```

Comment:
Let's avoid default implementation for auto-generated code, it is not really needed

Reply:
Why not? It is kinda useful and simplifies things a little.

Btw, we can put here smth like `EMPTY_ARRAY` instead to avoid allocations

Reply:
Was thinking about excessive bytecode, but I was wrong.
Yeah, it would be nice to have `EMPTY_ARRAY`

Reply:
Unfortunately, we can't save bytecode here. It would be either `DefaultInpls.typeParamSerializers()` or just generated `return arrayOf()`